### DISCUSSION TITLE
docker 部署的ddns-go  忘记用户名密码，怎么重置？

### DISCUSSION THREAD

**Discussion by piaoliuchong** [jeessy2/ddns-go#1082] [General]
通过docker  run部署的ddns-go 忘记用户名密码，一直的登录失败，通过docker exe -it 容器号  rm  ddns-go删除了文件，还是要求用户名密码，删除容易，重建，也是用户名密码，那个配置文件在哪放着？

**Comment by jeessy2** [comment]
在docker的/root目录，ls -al

**Reply by piaoliuchong** [reply]
ls -al 看到三个灰色root，一个绿色ddns-go ，rm可以删除ddns-go，但是不能删除root。

现在解决方案，删除容器，重新docker run，改成ddns-go1就可以重新配置了

**Reply by xuanxuan1231** [reply]
猜测：删的时候可能要连着存储卷一起删
